ga4gh/cli.py

```diff
@@ -1531,6 +1532,15 @@ def __init__(self, args):
         self._repoPath = args.repoPath
         self._repo = datarepo.SqlDataRepository(self._repoPath)
 
+    def _getDatasetFromArgs(self):
+        try:
```

Comment:
This isn't necessary --- the exception should be propagated up automatically and caught at the CLI. This should be working nicely for (say) ReadGroupSets in the current sql repo code.